packages/@react-spectrum/s2/src/Button.tsx

```diff
@@ -430,6 +430,7 @@ export const LinkButton = forwardRef(function LinkButton(props: LinkButtonProps,
   props = useFormProps(props);
   let domRef = useFocusableRef(ref);
   let overlayTriggerState = useContext(OverlayTriggerStateContext);
+  let variant = props.variant || 'primary';
```

Comment:
move to this format instead?
```suggestion
  let {
    fillStyle,
    variant = 'primary',
  } = props;
```
otherwise it should be `let variant = props.variant ?? 'primary'`